boards/arc/nsim/board.cmake

```diff
@@ -9,6 +9,9 @@ endif()
 if(${CONFIG_SOC_NSIM_EM})
 board_runner_args(arc-nsim "--props=nsim_em.props")
 board_runner_args(mdb "--nsim_args=mdb_em.args")
+elseif(${CONFIG_SOC_NSIM_EM7D_V22})
+board_runner_args(arc-nsim "--props=nsim_em7d_v22.props")
+board_runner_args(mdb "--nsim_args=mdb_em7d_v22.args")
```

Comment:
I know you just follow the existing style, but it would be appreciated if you add an extra commit that corrects the indentaion in the already existing `if` statements.

```
f(${CONFIG_SOC_NSIM_EM})
  board_runner_args(arc-nsim "--props=nsim_em.props")
  board_runner_args(mdb "--nsim_args=mdb_em.args")
elseif(${CONFIG_SOC_NSIM_EM7D_V22})
  board_runner_args(arc-nsim "--props=nsim_em7d_v22.props")
  board_runner_args(mdb "--nsim_args=mdb_em7d_v22.args")
elseif(${CONFIG_SOC_NSIM_SEM})
  board_runner_args(arc-nsim "--props=nsim_sem.props")
  board_runner_args(mdb "--nsim_args=mdb_sem.args")
...
```
and so on.